app/flatpak-builtins-remote-delete.c

```diff
@@ -104,9 +104,9 @@ flatpak_builtin_remote_delete (int argc, char **argv, GCancellable *cancellable,
           if (transaction == NULL)
             return FALSE;
 
-          for (i = 0; refs[i]; i++)
+          for (i = 0; g_ptr_array_index (refs_to_remove, i); i++)
```

Comment:
it would be more idiomatic to make this

```
for (i = 0; i < refs_to_remove->len; i++)
  {
    ref = g_ptr_array_index (refs_to_remove, i);
   ....
```

Reply:
sure